presto-docs/src/main/sphinx/optimizer/cost-based-optimizations.rst

```diff
@@ -0,0 +1,76 @@
+========================
+Cost based optimizations
+========================
+
+Presto supports several cost based optimizations, described below.
+
+Join Enumeration
+----------------
+
+The order in which joins are executed in a query can have a significant impact
+on the query's performance. The aspect of join ordering that has the largest
+impact on performance is the size of the data being processed and transferred
+over the network. If a join that produces a lot of data is performed early in
+the execution, then subsequent stages will need to process large amounts of
+data for longer than necessary, increasing the time and resources needed for
+the query.
+
+With cost based join enumeration, Presto uses
+:doc:`/optimizer/statistics` provided by connectors to estimate
+the costs for different join orders and automatically pick the
+join order with the lowest computed costs.
+
+The join enumeration strategy is governed by the ``join_reordering_strategy``
+session property, with the ``optimizer.join-reordering-strategy``
+configuration property providing the default value.
+
+The valid values are:
+ * ``AUTOMATIC`` (default) - full automatic join enumeration enabled
```

Comment:
It is not a "default" just yet, but will be soon: https://github.com/prestodb/presto/pull/12752

Reply:
yeah, i would change this for now to mark ELIMINATE_CROSS_JOINS as the default and update the documentation in #12752

Reply:
Changed to reflect correct default of ELIMINATE_CROSS_JOINS